editoast/osm_to_railjson/justfile

```
@@ -0,0 +1,30 @@
+#!/usr/bin/env just --justfile
+
+# Display available recipes
+help:
+    @just --list --unsorted
+
+# Run the application; does not start other services
+run *args:
+    cargo run {{args}}
+
+# Install required dependencies
+install:
+    cargo build
+
+# Run the tests
+test:
+    cargo test
+
+# Format the source code
+format:
+    cargo fmt --all
+    taplo fmt
+
+# Verify code lints without applying any fix
+lint:
+    cargo clippy --all-features --all-targets
```

nitpick: for coherence with other Rust projects.

```suggestion
    cargo clippy --workspace --all-features --all-targets
```